OSDOCS-5851-Newer: Document post-install AWS LZ work (#65877)
openshift/openshift-docs: modules/nw-cluster-mtu-change.adoc

// Module included in the following assemblies:
//
// * networking/changing-cluster-network-mtu.adoc
// * post_installation_configuration/aws-compute-edge-tasks.adoc

ifeval::["{context}" == "aws-compute-edge-tasks"]
:localzone:
endif::[]

:_content-type: PROCEDURE
[id="nw-cluster-mtu-change_{context}"]
= Changing the cluster MTU

> 

Reviewer: Might wanna remove this extra space

.Verification

ifdef::localzone[]

> * You can verify that a node in your cluster uses an MTU that you specified in the previous procedure. To get the current MTU for the cluster network, enter the following command:

Reviewer: Maybe we could condense this? or not. it's just a suggestion.

```suggestion
* Verify that the node in your cluster uses the MTU that you specified in the previous procedure by entering the following command:
```
